modules/network-observability-virtualization-configuration.adoc

```diff
@@ -0,0 +1,93 @@
+
+.Procedure
+. Get information about the virtual machine launcher pod by running the following command. This information is used in Step 5: 
++
+[source,terminal]
+----
+$ oc get pod virt-launcher-<vm_name>-<suffix> -n <namespace> -o yaml
+----
++
+[source,yaml]
+----
+$ oc get pod virt-launcher-fedora-aqua-fowl-13-zr2x9 -n my-vms -o yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    k8s.v1.cni.cncf.io/network-status: |- 
+      [{
+        "name": "ovn-kubernetes",
+        "interface": "eth0",
+        "ips": [
+          "10.129.2.39"
+        ],
+        "mac": "0a:58:0a:81:02:27",
+        "default": true,
+        "dns": {}
+      },
+      {
+        "name": "my-vms/l2-network",   <1>
+        "interface": "podc0f69e19ba2", <2>
+        "ips": [
+          "10.10.10.15"
+        ], <3>
+        "mac": "02:fb:f8:00:00:12",    <4>
+        "dns": {}
+      }]
+  name: virt-launcher-fedora-aqua-fowl-13-zr2x9
+  namespace: my-vms
+spec:
+  ...
+status:
+  ...
+----
+<1> The name of the secondary network.
+<2> The network interface name of the secondary network.
+<3> The list of IPs used by the secondary network.
+<4> The MAC address used for secondary network.
+
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. Under the *Provided APIs* heading for the *NetObserv Operator*, select *Flow Collector*.
+. Select *cluster* and then select the *YAML* tab.
+. Configure `FlowCollector` based on the information you found from the additional network investigation. 
++
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+# ...
+ebpf:
+    privileged: true              <1>
+  processor:
+    advanced:
+      secondaryNetworks:
+      - index:                    <2>
+        - Interface
+        - MAC                     <3>
+        - IP                      <4>
+        name: my-vms/l2-network   <5> 
+# ...
+----
+<.> Ensure that the ebpf agent is in `privileged` mode so that the flows are enriched according to the MAC address.
+<.> Define the fields to use for indexing the virtual machine launcher pods. These should form a unique identifier across the cluster according to the fields available in `k8s.v1.cni.cncf.io/network-status` annotation. Not all secondary networks have all the index fields in this example. Ensure that only the fields that are annotated in your secondary interface are listed here and if they are not annotated, they are removed.
+<.> If your additional network information has a MAC address, specify it here.
+<.> If your additional network information has an IP address, specify it here.
```

Comment:
```suggestion
<.> If your additional network information has a MAC address, add `MAC` to the field list.
<.> If your additional network information has an IP address, add `IP` to the field list.
```

The `specify it here` mentions are confusing to me. I feel it requires to fill the values found in the pod annotation instead of the field names.

Reply:
Makes sense. Added your suggestion